site/content/en/blog/releases/1.40.0.md

```diff
@@ -0,0 +1,74 @@
+
+In this release, we’ve updated our Kuberenetes support to include version 1.29, introduced a new implementation in the CSharp SDK for the Counters and Lists, graduated the FleetAllocationOverflow feature to stable, moved the DisableResyncOnSDKServer feature into beta and refined the documentation for Counters and Lists to simplify usage and enhance clarity.
+
+- **Kubernetes Support Update 1.29**: We’ve upgraded our Kubernetes compatibility to include version 1.29 and have discontinued support for version 1.26. Our current supported versions are now 1.27, 1.28, and 1.29. Please ensure your clusters are upgraded to one of these supported versions.
+
+- **CSharp SDK for Counters and Lists**: We’ve rolled out a new implementation in the CSharp SDK for the Counters and Lists features, along with significant enhancements to the documentation. These updates simplify the process of tracking numeric counters and lists of values on GameServers, providing clearer guidance.
+
+- **Support for Extended Duration Pods on GKE Autopilot**: Starting with Kubernetes version 1.28, we now support extended duration pods in Agones on GKE Autopilot. The GKEAutopilotExtendedDurationPods feature has been moved to Alpha status and requires Kubernetes versions 1.28 or newer. This enhancement is essential for those needing longer-lived pods in managed Kubernetes environments.
+
+- **DisableResyncOnSDKServer in Beta**: The DisableResyncOnSDKServer feature has progressed to beta in this release and is now enabled by default. This feature sets the resync interval to 0 to enhance performance by eliminating unnecessary resyncs.
+
+- **FleetAllocationOverflow now Stable**: With the FleetAllocationOverflow feature now stable in this release, you can apply labels and/or annotations to GameServers that exceed the desired number of replicas in a GameServerSet. This occurs when the allocation of GameServers surpasses the specified replica count.
+
+- **Unity SDK Update: Enhanced Address Retrieval for GameServers**: We've enhanced functionality to allow developers to conveniently access all associated addresses, including both IP and DNS, for a game server. This change simplifies the integration and deployment process by providing complete address information.
+
+- **Counters & Lists Documentation Enhancements**: We've standardized sorting priorities in GameServerAllocation and Fleet to consistently use available capacity and set the default sorting order as Ascending, making it easier to understand. We also have a new Counters and Lists [documentation landing page](https://agones.dev/site/docs/guides/counters-and-lists/) to explain the functionality in full.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.40.0" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Breaking changes:**
+- Optimise GameServer Sub-Controller Queues by @markmandel in https://github.com/googleforgames/agones/pull/3781
```

Comment:
This isn't a breaking change -- breaking change is a break in an API surface.

This can go down under **Implemented enhancements:** (even though it was technically a bug fix).

Otherwise, this LGTM.